src/Libraries/RevitNodes/Elements/Group.cs

```diff
@@ -0,0 +1,191 @@
+        /// <returns>The new group type</returns>
+        public static Group FromElements(List<Element> elements)
+        {
+            List<ElementId> elementIds = elements.Select(x => x.InternalElement.Id).ToList();
+            TransactionManager.Instance.EnsureInTransaction(Document);
+            var group = Document.Create.NewGroup(elementIds).ToDSType(true) as Group;
+            TransactionManager.Instance.TransactionTaskDone();
+            return group;
+        }
+
+        /// <summary>
+        /// Place an instance of a Model Group into the Autodesk Revit document, using a location and a group type.
+        /// </summary>
+        /// <param name="location">The point to place the group.</param>
+        /// <param name="groupType">The type of group to place.</param>
+        /// <returns>The new group instance.</returns>
+        public static Group PlaceInstance(Autodesk.DesignScript.Geometry.Point location, Element groupType)
+        {
+            var internalType = groupType.InternalElement as Autodesk.Revit.DB.GroupType;
+            if (internalType == null)
+                throw new InvalidOperationException(String.Format(Properties.Resources.InvalidGroupType, nameof(groupType)));
+
+            XYZ newLocation = location.ToRevitType(true);
+            TransactionManager.Instance.EnsureInTransaction(Document);
+            Group newGroup = Document.Create.PlaceGroup(newLocation, internalType).ToDSType(true) as Group;
+            TransactionManager.Instance.TransactionTaskDone();
+
+            return newGroup;
+        }
+
+        #endregion
+
+        #region public properties
+
+        /// <summary>
+        /// 
+        /// </summary>
+        public Autodesk.DesignScript.Geometry.Point Location
+        {
+            get 
+            {
+                if (this.InternalGroup.IsAttached)
+                    throw new InvalidOperationException(Properties.Resources.AttachedGroupLocation);
+
+                var locationPt = this.InternalGroup.Location as LocationPoint;
+                return locationPt.Point.ToPoint();
+            }
+        }
+
+        /// <summary>
+        /// 
+        /// </summary>
+        public Element GroupType
+        {
+            get { return this.InternalGroup.GroupType.ToDSType(true); }
+        }
+
+        /// <summary>
+        /// Indicates wether or not this group is attached to a parent group.
+        /// </summary>
+        public bool IsAttached
+        {
+            get { return this.InternalGroup.IsAttached; }
+        }
+
+        /// <summary>
+        /// Returns the attached detail groups available for this group.
+        /// </summary>
+        public List<Element> AttachedDetailGroup
+        {
+            get 
+            {
+                var attachedGroupId = this.InternalGroup.GetAvailableAttachedDetailGroupTypeIds();
+                return attachedGroupId.Select(id => Document.GetElement(id).ToDSType(true)).ToList();
```

Comment:
if there aren't any, `Select` will return `null` and then will `ToList()` throw ? If it'll throw, the error message might be not understandable to users. If it doesn't throw and just returns `null`, then fine.

Reply:
if there arent any it will return an empty list
![image](https://user-images.githubusercontent.com/13732445/72820744-53982780-3c67-11ea-9039-5de83f4834fc.png)